docs/src/api/class-response.md

```diff
@@ -58,7 +58,7 @@ Name of the header.
 
 ## async method: Response.json
 * langs: js, python
-- returns: <[Serializable]>
```

Comment:
Would it make sense to change `export type Serializable = {};` to `export type Serializable = any;` ?

Reply:
Yes I agree. I don't want to lose the concept of Serializable in the docs.